homeassistant/components/ambiclimate/strings.json

```diff
@@ -7,14 +7,14 @@
       }
     },
     "create_entry": {
-      "default": "Successfully authenticated with Ambiclimate"
+      "default": "[%key:common::config_flow::create_entry::authenticated%]"
     },
     "error": {
       "no_token": "Not authenticated with Ambiclimate",
       "follow_link": "Please follow the link and authenticate before pressing Submit"
     },
     "abort": {
-      "already_configured_account": "[%key:common::config_flow::abort::already_configured_account%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
```

Comment:
Why go to the bother of renaming all the keys if the final common string you are pointing to is staying as already_configured_account?

Reply:
https://github.com/home-assistant/core/issues/40578#issuecomment-707146513
https://github.com/home-assistant/core/issues/40578#issuecomment-707250137